Ex_JDBC01/src/main/java/lt/verbus/config/MySqlConfig.java

```diff
@@ -0,0 +1,9 @@
+package lt.verbus.config;
+
+public class MySqlConfig {
+
+    public static final String SQL_URL = "jdbc:mysql://localhost:3306/employees";
+    public static final String SQL_USERNAME = "root";
+    public static final String SQL_PASSWORD = "08princeAli";
```

Comment:
Keistai atrodo public static password :) As loadinciau is .properties failo

Reply:
Sutapimo lygis: prieš kelias minutes Udemy kurse būtent apie properties išmokau. Bet čia Springe per @Value